.coveragerc

```diff
@@ -14,25 +14,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Generated by synthtool. DO NOT EDIT!
```

Comment:
This is really a comment for the generator and not this particular PR, but in general, I'm in favor of clearly marking files which are generated. (I'm assuming this file is still generated.) The less detective work you can make a developer do, the better.

Reply:
This goes for everything under `services` and `types`, too. Or maybe it could be a README in those folders. But a clear delineation between generated and hand written code, I think, is helpful.